Modules/_ctypes/callproc.c

```diff
@@ -1944,23 +1963,37 @@ POINTER(PyObject *self, PyObject *cls)
     return result;
 }
 
+/*[clinic input]
+_ctypes.pointer as create_pointer_inst
+
+    obj: object
```

Comment:
You can do the same trick here :)
`obj as arg: object`

Reply:
I thought I'd actually rename it here since `obj` seems like a more descriptive name and it doesn't clash with another variable in this instance. What do you think?

Reply:
There is a little convention about C-level argument name within this file: most one-argument functions use `obj`, but `buffer_info`, `My_Py_INCREF` and `My_Py_DECREF` use `arg`. I think it isn't essential for such small function, and the old name doesn't affect readability this much. In the end, the less diff the better :)

Reply:
Point taken about keeping the diff small. I applied your suggestion :)